### PR TITLE
Accept empty domain 

### PR DESCRIPTION
**Pull request by mchf**
Bnc#900647

### CODE REVIEW

File: package/yast2-network.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Oct  6 16:44:44 UTC 2015 - mfilka@suse.com
+
+- bnc#900647
+  - accepts empty domain for /etc/hostname
```
Comment (mvidner) [inline package/yast2-network.changes]: > Fixed to accept an empty domain in /etc/hostname

or

> Fixed to require a domain in /etc/hostname

?

**Comment by mvidner** [comment]
Just clarify the changelog message, otherwise LGTM

File: package/yast2-network.changes (hunk 1 above)
Comment (mchf) [inline package/yast2-network.changes]: > Fixed to accept an empty domain in /etc/hostname

this one

**Comment by mchf** [comment]
Updated changelog, merging once passes CI checks. Thanks for review

**Pull request merged**